### DISCUSSION TITLE
What is the lowest Python version are you using?

### DISCUSSION THREAD

**Discussion by lewisblake** [metno/pyaerocom#840] [Polls]
We are trying to determine when to phase out support for Python 3.8. What is the lowest Python version you are using?

**Comment by avaldebe** [comment]
For reference this is the `numpy` doping support schedule for different versions of Python [NEP29]:
- 2023-04-14 drop support for Python 3.8
- 2024-04-04 drop support for Python 3.9
- 2025-04-04 drop support for Python 3.10
- 2026-04-24 drop support for Python 3.11

[NEP29]: https://numpy.org/neps/nep-0029-deprecation_policy.html#drop-schedule
